File: ConcursInotCS/ConcursInotCS/domain/Proba.cs
namespace ConcursInotCS.domain;

public class Proba
{
    private int distanta;
    private string stil;

    public Proba(int distanta, string stil)
    {
        this.distanta = distanta;
        this.stil = stil;
    }

    public int Distanta
    {
        get => distanta;
        set => distanta = value;
    }

    public string Stil
    {
        get => stil;
        set => stil = value ?? throw new ArgumentNullException(nameof(value));
    }
}

File: ConcursInot/src/main/java/ro/mpp2024/domain/Proba.java
package ro.mpp2024.domain;

public class Proba extends Entity<Integer>{
    private int distanta;
    private String stil;

    public Proba(int distanta, String stil) {
        this.distanta = distanta;
        this.stil = stil;
    }

    public int getDistanta() {
        return distanta;
    }

    public void setDistanta(int distanta) {
        this.distanta = distanta;
    }

    public String getStil() {
        return stil;
    }

    public void setStil(String stil) {
        this.stil = stil;
    }
}
